consensus/src/types/mod.rs

```diff
@@ -323,3 +323,11 @@ impl From<&OptimisticUpdate> for GenericUpdate {
         }
     }
 }
+/// Aggregate all update types that are needed to verify and apply a new Ethereum state.
```

Comment:
```suggestion
/// Holds an aggregate of all update types that are needed to verify and apply a new Ethereum state.
```
When describing a struct, you should use different grammar than when describing a function. This grammar is more suitable for the function that creates this struct. But for this struct, you should describe what it holds rather than what "it does" (it doesn't do anything)